src/InnerApi/Controllers/PriceHistoryController.cs

```diff
@@ -34,13 +34,21 @@ public PriceHistoryController(IQueryDispatcher queryDispatcher, ICommandDispatch
         /// Create apprenticeship price change
         /// </summary>
         /// <param name="apprenticeshipKey">The unique identifier of the apprenticeship</param>
-        /// <param name="request">Details of the requested price change</param>
-        /// <returns>Ok result</returns>
+        /// <param name="request">Details of the requested price change. NB one of either employer or provider id should be included to indicate the initiator of the request</param>
+        /// <returns>Ok on success, Bad Request if both or neither employer & provider id are provided</returns>
```

Comment:
using an ampersand causes a build warning 'Warning	CS1570	XML comment has badly formed XML'

Reply:
good spot this is also out of date given the design change we made, will update